templates/registry/markdown/stability.j2

```diff
@@ -4,6 +4,8 @@
 {%- elif stability == "stable" %}![Stable](https://img.shields.io/badge/-stable-lightgreen)
 {%- elif stability == "release_candidate" %}![Release Candidate](https://img.shields.io/badge/-rc-mediumorchid)
 {%- elif stability == "deprecated" %}![Deprecated](https://img.shields.io/badge/-deprecated-red)
-{%- else %}![Experimental](https://img.shields.io/badge/-experimental-blue)
+{%- elif stability == "experimental" %}![Experimental](https://img.shields.io/badge/-experimental-blue)
+{%- elif stability == "development" %}![Experimental](https://img.shields.io/badge/-experimental-blue)
```

Comment:
we should change this one to `development`, keeping it as  `Experimental` to minimize diff